packages/blade/src/components/Tag/_decisions/decisions.md

```diff
@@ -0,0 +1,208 @@
+# Tag 🏷️
+
+These are set of interactive keywords that help organise & categorise objects. Tags can be added or removed from an object by the users.
+
+<img width="426" alt="image" src="https://github.com/razorpay/blade/assets/30949385/75d8068d-d6fc-47e7-aa75-21686ed1c27d">
+
+## Design
+
+- [Tags - Figma Design](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?type=design&node-id=29210-567576&t=HLRjz3OTofBtFdDl-0)
+- [Tags - Motion](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?type=design&node-id=29744-568562&mode=design&t=KI2KRJMzUpz8Ymr8-0)
+
+## Proposed API
+
+```jsx
+<Tag
+  icon={CheckIcon}
+  size="medium"
+  onDismiss={({ value }) => {
+    console.log('Close Icon Clicked', value);
+  }}
+>
+  Unpaid
+</Tag>
+```
+
+### Props
+
+```ts
+type TagProps = {
+  /**
+   * Decides the size of Tag
+   *
+   * @default medium
+   */
+  size?: 'medium' | 'large';
+
+  /**
+   * Any icon from Blade
+   */
+  icon?: IconComponent;
```

Comment:
No option of icon position?

Reply:
No. Tag has close icon button on trailing so any icon that is added on tag will always on leading part only.